MountList.lua

```diff
@@ -191,6 +191,33 @@ function LM.MountList:PriorityRandom(r)
 end
 
 function LM.MountList:RarityRandom(r)
+    if #self == 0 then return end
+    if not MountsRarityAddon or not MountsRarityAddon.MountsRarity then
+        LM.Debug(" - RarityRandom: MountsRarity not loaded. Defaulting to SimpleRandom.")
+        return self:SimpleRandom(r)
+    end
```

Comment:
Again if we go down the LibStub library route we won't need to worry about this as LiteMount will always have a copy.